docs/reference/sailing/miscellaneous.md

```diff
@@ -0,0 +1,95 @@
+# Miscellaneous
```

Comment:
Maybe specify that this is Miscellaneous Sailing Knowledge

Reply:
👍 Next commit will address this